lesson1/script.js

```diff
@@ -0,0 +1,2 @@
+var list = document.getElementById("js-lists");
```

Comment:
you can use `const`.
`var` will not be used anymore.

Reply:
I understand that `var` is not used in the current script.